jep/1/README.adoc

```diff
@@ -185,25 +184,25 @@ For simplicity, this document uses the singular
 ("The JEP Sponsor", "a sponsor")
 when referring the one or more people in the role of "Sponsor" of a JEP.
 
-==== Reviewer
+==== BDFL Delegate
 
-The JEP Reviewer is a contributor who will make the final decision whether to accept a JEP.
-The <<BDFL>> is the de facto reviewer for all JEPs, but may delegate the role to someone else.
+The <<BDFL>> is the final authority for reviewing and resolving JEPs,
+but may delegate that responsibility to another contributor,
+a "BDFL Delegate" on a per-JEP basis.
 
-Whenever a new JEP is submitted, any core contributor that believes they are suitably
-experienced to make the final decision on that JEP may offer to serve as the "Reviewer" for that JEP.
-A JEP's <<Sponsor>> cannot also be its Reviewer.
+A BDFL Delegate may be selected at any point before the JEP is reviewed.
+Any contributor with sufficient technical and organizational
+experience to make the final decision on that JEP,
+may offer to be the BDFL's Delegate for a JEP.
 If their self-nomination is accepted by the other core contributors and the BDFL,
 then that contributor will have the authority to accept (or reject) that JEP.
-The BDFL may also choose to be the Reviewer for a JEP, overriding any self-nominations.
-If no other contributor volunteers, or if the core contributors (including the BDFL)
-cannot agree on a suitable Reviwer, the role reverts to the BDFL.
+The BDFL Delegate for a JEP will be recorded in the
+<<header-delegate, "BDFL-Delegate" header>> in the JEP.
 
-If a contributor other than the BDFL will be the Reviewer for the JEP,
-they should be selected before the JEP is submitted.
+A JEP's <<Sponsor>> cannot also be the BDFL Delegate for that JEP.
```

Comment:
Does this mean if I ask Oleg to be the Delegate for the remoting space in general, Oleg cannot sponsor a JEP to make a change in remoting?

Reply:
FWIW, I want Oleg to be able to sponsor a change in remoting, drive toward consensus, and accept that JEP himself.

Reply:
@kohsuke - fair point.